### DISCUSSION TITLE
sorry, unimplemented: 64-bit mode not compiled in

### DISCUSSION THREAD

**Discussion by rifting** [pocketbase/pocketbase#2708] [Q&A]
When running `go build` I get the error

# github.com/mattn/go-sqlite3
cc1.exe: sorry, unimplemented: 64-bit mode not compiled in

**Comment by ganigeorgiev** [comment]
Hm, it seems that you have cgo enabled but your platform or the installed gcc compiler doesn't seem compatible (or need further configuration).

There are not enough details about your setup to understand or reproduce the issue, but to avoid platform dependency issues, you can try building your app with `CGO_ENABLED=0`, eg.:

```sh
CGO_ENABLED=0 go build
```

**Comment by rifting** [comment]
I'm running on windows and as such cannot do `CGO_ENABLED=0 go build`

**Reply by ganigeorgiev** [reply]
`CGO_ENABLED` is environment variable. If the shell you are using doesn't allow defining one-off env vars, then you have to search how to define it globally for your system.

Alternatively, you can also try with the builtin `go env` command:

```sh
# change the default CGO_ENABLED value
$ go env -w CGO_ENABLED=0

# ensure that the change was applied
$ go env CGO_ENABLED

# run build in your app directory
$ go build
```

**Comment by rifting** [comment]
This fixed the issue. Thank you.
